src/frontend/components/Felleskomponenter/Steg/Steg.tsx

```diff
@@ -167,78 +146,71 @@ function Steg({ tittel, guide, skjema, gåVidereCallback, vedleggOppsummering, c
-                    </Heading>
-                </Box>
-                {guide && (
-                    <Box marginBlock="0 12">
-                        <GuidePanel poster>{guide}</GuidePanel>
-                    </Box>
+                                <ArrowLeftIcon aria-hidden />
+                                {plainTekst(tilbakeKnapp)}
+                            </Link>
+                        </div>
+                        <Box paddingBlock="6 5">
+                            <Heading level="2" size={'large'}>
+                                {tittel}
+                            </Heading>
+                        </Box>
+                        <FormProgress
+                            translations={{
+                                step: formProgressStegOppsummeringTekst,
+                                showAllSteps: plainTekst(frittståendeOrdTekster.visAlleSteg),
+                                hideAllSteps: plainTekst(frittståendeOrdTekster.skjulAlleSteg),
+                            }}
+                            totalSteps={formProgressSteg.length}
+                            activeStep={hentNåværendeStegIndex()}
+                            onStepChange={stegIndex => håndterGåTilSteg(stegIndex - 1)}
+                        >
+                            {formProgressSteg.map((value, index) => (
+                                <FormProgress.Step
+                                    key={index}
+                                    completed={index + 1 < hentNåværendeStegIndex()}
+                                    interactive={index + 1 < hentNåværendeStegIndex()}
+                                >
+                                    {value.tittel}
+                                </FormProgress.Step>
+                            ))}
+                        </FormProgress>
+                    </div>
                 )}
-                <Form onSubmit={event => håndterGåVidere(event)} autoComplete="off">
-                    <ChildrenContainer>{children}</ChildrenContainer>
-                    {skjema && visFeiloppsummering(skjema.skjema) && (
-                        <SkjemaFeiloppsummering skjema={skjema.skjema} />
-                    )}
-                    {visVedleggOppsummering && (
-                        <Alert variant="info">
-                            {plainTekst(dokumentasjonTekster.lastOppSenereISoknad)}
-                            <VedleggOppsummering vedlegg={vedleggOppsummering} />
-                        </Alert>
-                    )}
-                    {!erPåKvitteringsside() && (
-                        <Navigeringspanel
-                            onTilbakeCallback={håndterTilbake}
-                            onAvbrytCallback={håndterAvbryt}
-                            valideringErOk={skjema && skjema.valideringErOk}
-                        />
-                    )}
-                </Form>
+                {guide && <GuidePanel poster>{guide}</GuidePanel>}
```

Comment:
Kun et spørsmål: Jeg ser at guide her har endret rekkefølge, nå ligger det etter form progress. Det er mening sant?

Reply:
Ja, dette er meningen slik at det følger [malen/eksempelet til Aksel](https://aksel.nav.no/monster-maler/soknadsdialog/oppsummeringsside-for-soknadsdialoger).